pkgs/applications/virtualization/docker/default.nix

```diff
@@ -64,16 +59,73 @@ rec {
 
       NIX_CFLAGS_COMPILE = "-DMINIMAL=ON";
     });
+
+    moby = buildGoPackage ((optionalAttrs (stdenv.isLinux)) rec {
+      name = "moby-${version}";
+      inherit version;
+      inherit docker-runc docker-containerd docker-proxy docker-tini;
+
+      src = fetchFromGitHub {
+        owner = "moby";
+        repo = "moby";
+        rev = mobyRev;
+        sha256 = mobySha256;
+      };
+
+      goPackagePath = "github.com/docker/docker";
+
+      nativeBuildInputs = [ pkgconfig go-md2man go libtool installShellFiles ];
+      buildInputs = [
+        makeWrapper sqlite lvm2 btrfs-progs systemd libseccomp
+      ];
+
+      extraPath = optionals (stdenv.isLinux) (makeBinPath [ iproute iptables e2fsprogs xz xfsprogs procps util-linux git ]);
+
+      dontStrip = true;
```

Comment:
Why are disabling this?

Reply:
Probably unnecessary now - looking at [other Linux distribution packaging script](https://github.com/archlinux/svntogit-community/commit/ff77881c461d895472a91f4f04da43b878bf58e5#diff-37538beb61ff63edebbf735dfcf39e5d732f49183d6beb097169d971875ca422L17), then they dropped it as well :thinking:

`# don't strip binaries! A sha1 is used to check binary consistency.` - I wonder what this really means.

Reply:
I guess the binary might check its own hash during runtime or something? :thinking: 
Anyways, I checked that it works without this line, so can we remove it?

Reply:
It's already removed - see latest changes. I just left conversation in unresolved stare.